lib/auth/grpcserver.go

```diff
@@ -2356,7 +2356,9 @@ func userSingleUseCertsAuthChallenge(gctx *grpcContext, stream proto.AuthService
 	return mfaDev, nil
 }
 
-func userSingleUseCertsGenerate(ctx context.Context, actx *grpcContext, req proto.UserCertsRequest, mfaDev *types.MFADevice) (*proto.SingleUseUserCert, error) {
+// mfaVerifiedSessionExpires is the expiry of the user cert from which this single use cert is derived from.
```

Comment:
Is this comment misplaced?

Reply:
I think I was trying to document what the `previousIdentityExpires` parameter does, but that's documented enough elsewhere that I'll just get rid of it.